ci/test-conformance-eks.sh

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -exu
+
+function echoerr {
+    >&2 echo "$@"
+}
+
+CLUSTER="${JOB_NAME}-${BUILD_NUMBER}"
+REGION="us-east-2"
+K8S_VERSION="1.15"
+AWS_NODE_TYPE="t3.medium"
+SSH_KEY_PATH="/home/ubuntu/.ssh/id_rsa.pub"
+
+_usage="Usage: $0 [--cluster-name] [--aws-access-key] [--aws-secret-key] [--aws-region] [--ssh-key]"
+
+function print_usage {
+    echoerr "$_usage"
+}
+
+function print_help {
+    echoerr "Try '$0 --help' for more information."
+}
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+
+case $key in
+    --cluster-name)
+    CLUSTER="$2"
+    shift 2
+    ;;
+    --aws-access-key)
+    AWS_ACCESS_KEY="$2"
+    shift 2
+    ;;
+    --aws-secret-key)
+    AWS_SECRET_KEY="$2"
+    shift 2
+    ;;
+    --aws-region)
+    REGION="$2"
+    shift 2
+    ;;
+    --ssh-key)
+    SSH_KEY_PATH="$2"
+    shift 2
+    ;;
+    -h|--help)
+    print_usage
+    exit 0
+    ;;
+    *)    # unknown option
+    echoerr "Unknown option $1"
+    exit 1
+    ;;
+esac
+done
+
+function setup_eks() {
+
+    echo "=== This cluster to be created is named: ${CLUSTER} ==="
+    echo "CLUSTERNAME=${CLUSTER}" > ci_properties.txt
+
+    echo "=== Using the following awscli version ==="
+    aws --version
+
+    set +e
+    aws configure << EOF
+${AWS_ACCESS_KEY}
+${AWS_SECRET_KEY}
+${REGION}
+JSON
+EOF
+    set -e
+    printf "\n"
+    echo "=== Using the following kubectl ==="
+    which kubectl
+
+    echo '=== Creating a cluster in EKS ==='
+    eksctl create cluster \
+      --name ${CLUSTER} --region ${REGION} --version=${K8S_VERSION} \
+      --nodegroup-name workers --node-type ${AWS_NODE_TYPE} --nodes 2 \
+      --ssh-access --ssh-public-key ${SSH_KEY_PATH} \
+      --managed
+    if [[ $? -ne 0 ]]; then
+        echo "=== Failed to deploy EKS cluster! ==="
+        exit 1
+    fi
+
+    mkdir -p ${WORKSPACE}/jenkins/out/
```

Comment:
I think I left this comment before, but can we make `-kubeconfig` (kubeconfig destination file) a command-line parameter for the script? You can then pass the correct path for jenkins when you invoke the script from projects.yaml.

Reply:
Maybe I missed kubeconfig part. Will update that